unit_test/tstParticleList.hpp

```diff
@@ -38,6 +38,17 @@ struct Bar : Field::Matrix<double, 3, 3>
 //---------------------------------------------------------------------------//
 void linearAlgebraTest()
 {
+    // Get inputs for mesh.
+    std::ifstream stream( "uniform_mesh_test_1.json" );
```

Comment:
Looks like you missed using `Picasso::parse()` here.

Reply:
Missed those - fixed